src/liquid/components/ld-select/test/ld-select.a11y.ts

```diff
@@ -1,59 +0,0 @@
-import 'globals'
```

Comment:
So you're removing these tests completely, instead of migrating them, because they didn't work (well integrated) anyway?

Reply:
Yes, if we do not use them regularly they are useless. My hope is that we soon manage to get e2e tests running on CI reliably and then everything will be fine.